### PR TITLE
:sparkles: feat: new event creator

### PR DESCRIPTION
**Pull request by larwaa**
### Changes

- Revamp create event to make it mobile-friendly and step-based
- Fix a bugged default value for allowed grade years
- Minor theme adjustments
- Move event creation to /events/new rather than events/create-event

Can be tested locally by going to http://localhost:3000/events/new

### Screenshots
Screenshot 2022-10-30 at 19 42 37
Screenshot 2022-10-30 at 19 42 44
Screenshot 2022-10-30 at 19 42 50
Screenshot 2022-10-30 at 19 42 58

### CODE REVIEW

**Review by simensandhaug — COMMENTED** [review]
LGTM, however when you change the MUI styling, may that not impact other parts of the site aswell?

**Review by MagnusHafstad — COMMENTED** [review]
image
This is quite confusing as it is the small icon that allows you to choose when something starts might be a windows problem.

**Review by MagnusHafstad — APPROVED** [review]
See comment, might be somthing worth fixing before merging

**Pull request merged**
